src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -5668,4 +5668,16 @@
   <data name="WRN_DuplicateAnalyzerReference_Title" xml:space="preserve">
     <value>Analyzer reference specified multiple times</value>
   </data>
+  <data name="ERR_RequiredMemberMustBeSet" xml:space="preserve">
+    <value>Required member '{0}' must be set in the object initializer or attribute constructor.</value>
```

Comment:
>or attribute constructor [](http://example.com/codeflow?start=71&length=24)

This sounds confusing. Perhaps it should say "in attribute arguments"?  
>Attribute  ::=
 [  AttributeModifier  :  ]  SimpleTypeName  [  OpenParenthesis  [  AttributeArguments  ]  CloseParenthesis  ]